lib/kintsugi/apply_change_to_project.rb

```diff
@@ -306,7 +306,8 @@ def apply_change_to_component(parent_component, change_name, change, parent_chan
         component = child_component(parent_component, change_name)
```

Comment:
Don't you need to change line 305 as well?

Reply:
No, we only need to use the display names when adding array elements. change_path already includes the key under which the hash is.